tests/integration/test_vulnerability_detector/test_providers/test_download_feeds.py

```diff
@@ -88,19 +86,23 @@ def test_download_feeds(get_configuration, configure_environment, restart_module
     """
     check_apply_test({'test_download_feeds'}, get_configuration['tags'])
 
-    provider = get_configuration['metadata']['provider_name']
     os = (get_configuration['metadata']['os']).capitalize()
```

Comment:
```suggestion
    os = get_configuration['metadata']['os'].capitalize()
```

Reply:
Done in 77223ed24d8fff611947e817d180bf744720ed4c